### PR TITLE
[YUNIKORN-2384] YuniKorn 1.5.0 Release Notes

### PR DESCRIPTION
**Pull request by craigcondit**
### What is this PR for?
Add release notes for YuniKorn 1.5.0.

### What type of PR is it?
* [ ] - Bug Fix
* [ ] - Improvement
* [ ] - Feature
* [x] - Documentation
* [ ] - Hot Fix
* [ ] - Refactoring

### Todos
* [ ] - Task

### What is the Jira issue?
https://issues.apache.org/jira/browse/YUNIKORN-2384

### Questions:
* [ ] - The licenses files need update.
* [ ] - There is breaking changes for older versions.
* [ ] - It needs documentation.

### CODE REVIEW

**Comment by craigcondit** [comment]
Draft status, as release is not out yet. Specifically, we are missing the release date.

File: src/pages/release-announce/1.5.0.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+
+### REST API enhancement to node utilization
+
+The YuniKorn REST API has been enhanced to support the retrieval of
+node utilization statistics for multiple resource types in addition to
+the existing support for dominant resource type. This allows for more
+expressive visualizations in the YuniKorn Web UI. See
+[YUNIKORN-2257](https://issues.apache.org/jira/browse/YUNIKORN-2257) for
+details.
+
```
Comment (chia7712) [inline src/pages/release-announce/1.5.0.md:106]: how about adding https://issues.apache.org/jira/browse/YUNIKORN-2235 ? That brings new and useful restful APIs.

Comment (craigcondit) [reply src/pages/release-announce/1.5.0.md:106]: Good catch. Added in latest commit.

File: src/pages/release-announce/1.5.0.md [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+orchestration frameworks like Kubernetes for on-prem and on-cloud use cases.
+
+## Overview
+The Apache YuniKorn community has resolved 215 [JIRAs](https://issues.apache.org/jira/issues/?filter=12353274) in this release.
+
+Release manager: TingYao Huang
+
+Release date: 2024-03-01 (TBD)
+
+## Highlights
```
Comment (pbacsko) [inline src/pages/release-announce/1.5.0.md:37]: Event streaming is also an important enhancement: https://issues.apache.org/jira/browse/YUNIKORN-2115

Comment (craigcondit) [reply src/pages/release-announce/1.5.0.md:37]: Added in latest commit.

**Review by pbacsko — APPROVED** [review]
LGTM. So far so good.

**Review by chia7712 — APPROVED** [review]
LGTM

**Pull request closed without merging**